platforms/quorum/configuration/roles/create/constellation/tasks/main.yaml

```diff
@@ -3,13 +3,42 @@
   set_fact:
     enode_data_list: []
 
-# Get enode data
-- name: Get enode data for each node of the organization
+# Set node status
+- name: Set node status
+  set_fact:
+    node_status: existing
+  when: network.config.genesis | default('', true) | trim == ''
```

Comment:
should rather be based on "bootnode"(geth)

Reply:
The same check can be used as it is for ibft too.
Also, genesis block information will only be available in network.yaml if there is already a network in place and we are trying to add a new org.